.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
+    ignore:
+        # cannot be updated until the etcd library is updated
+      - dependency-name: "google.golang.org/grpc"
```

Comment:
Hubble doesn't have the etcd dependency and the vendored grpc module is up-to-date.
```suggestion
```

Reply:
OK, I've removed this.